### PR TITLE
Promote two-fer to core and sync to 1.2.0

### LINKED ISSUE #639: Curriculum: promote two-fer to core (round 2)
- [ ] sync `two-fer` with the problem specifications, add the `default` argument instead of calling with `''`.
- [ ] promote `two-fer` to core, right after `hello-world`

### PR DESCRIPTION
**Pull request by SleeplessByte**
And make sure we expect an optional argument, instead of passing in null, requiring boolean logic.

Fixes #639

### CODE REVIEW

File: exercises/two-fer/two-fer.spec.js [hunk 1]
```diff
@@ -1,18 +1,15 @@
-import { twoFer } from './two-fer';
+import { twoFer } from './two-fer.js'
```
Comment (tejasbubane) [inline exercises/two-fer/two-fer.spec.js]: Extension is not required.

Comment (SleeplessByte) [reply exercises/two-fer/two-fer.spec.js]: Oh right! Sorry -- that's automagic :)

**Comment by tejasbubane** [comment]
@SleeplessByte There seems to be a conflict here because of another PR.

**Comment by SleeplessByte** [comment]
I'll fix these conflicts right away

**Pull request merged**